File: P1/P1/Ejercicio3/Personaje.py
class Personaje:
    def __init__(self):
        self.armadura = None
        self.arma = None
        self.habilidad = None

    def __str__(self):
        return f"Personaje:\n Armadura: {self.armadura.dar_apariencia()}\n Arma: {self.arma}\n Habilidad: {self.habilidad}\n"

File: P2/ejercicio3/lib/personaje.dart
import 'package:ejercicio3/armadura.dart';

//Producto final

class Personaje {
  
  Armadura? armadura;
  String? arma;
  String? habilidad;

  Personaje({this.armadura, this.arma, this.habilidad});

  String mostrarPersonaje() {
    String armaduratxt = armadura!.darApariencia();
    return "Armadura: $armaduratxt, Arma: $arma, Habilidad: $habilidad";
  }

}

